### ISSUE TITLE
App crashes?

### ISSUE THREAD

**Issue by rhutikcodes** [jiusanzhou/flutter_notification_listener#3]
D/NotificationsListenerService(18300): service initialized
D/NotificationsListenerService(18300): on call start listener service
D/NotificationsListenerService(18300): get the lock
D/NotificationsListenerService(18300): init finished
D/NotificationsListenerService(18300): send event to flutter side immediately!
D/NotificationsListenerService(18300): send notification event: {infoText=null, showWhen=true, package_name=com.spotify.music, text=Daft Punk, title=Around the World, subText=Liked Songs, timestamp=1625167959053}
D/AndroidRuntime(18300): Shutting down VM
E/AndroidRuntime(18300): FATAL EXCEPTION: main
E/AndroidRuntime(18300): Process: com.example.notif, PID: 18300

**Comment by jiusanzhou** [comment]
Hi Rhutik, I can't reproduce this crash, would you mind to provide your Android version, I will try to find specified device and debug this in few days later.

**Comment by braxtondiggs** [comment]
I am getting the same error I think it's because of typing, the notification response isn't a string always. Here is my error:
```
D/NotificationsListenerService(15606): send event to flutter side immediately!
D/NotificationsListenerService(15606): send notification event: {infoText=null, showWhen=true, package_name=com.google.android.googlequicksearchbox, text=Clear, title=73° in New York, subText=null, bigText=Clear, timestamp=1625896740224}
D/AndroidRuntime(15606): Shutting down VM
E/AndroidRuntime(15606): FATAL EXCEPTION: main
E/AndroidRuntime(15606): Process: com.cybmit.nidoran, PID: 15606
E/AndroidRuntime(15606): java.lang.IllegalArgumentException: Unsupported value: Clear
E/AndroidRuntime(15606): 	at io.flutter.plugin.common.StandardMessageCodec.writeValue(StandardMessageCodec.java:276)
E/AndroidRuntime(15606): 	at io.flutter.plugin.common.StandardMessageCodec.writeValue(StandardMessageCodec.java:273)
E/AndroidRuntime(15606): 	at io.flutter.plugin.common.StandardMessageCodec.writeValue(StandardMessageCodec.java:265)
E/AndroidRuntime(15606): 	at io.flutter.plugin.common.StandardMethodCodec.encodeMethodCall(StandardMethodCodec.java:38)
E/AndroidRuntime(15606): 	at io.flutter.plugin.common.MethodChannel.invokeMethod(MethodChannel.java:99)
E/AndroidRuntime(15606): 	at io.flutter.plugin.common.MethodChannel.invokeMethod(MethodChannel.java:83)
E/AndroidRuntime(15606): 	at im.zoe.labs.flutter_notification_listener.NotificationsHandlerService.sendEvent(NotificationsHandlerService.kt:271)
E/AndroidRuntime(15606): 	at im.zoe.labs.flutter_notification_listener.NotificationsHandlerService.access$sendEvent(NotificationsHandlerService.kt:30)
E/AndroidRuntime(15606): 	at im.zoe.labs.flutter_notification_listener.NotificationsHandlerService$onNotificationPosted$$inlined$synchronized$lambda$1.run(NotificationsHandlerService.kt:144)
E/AndroidRuntime(15606): 	at android.os.Handler.handleCallback(Handler.java:938)
E/AndroidRuntime(15606): 	at android.os.Handler.dispatchMessage(Handler.java:99)
E/AndroidRuntime(15606): 	at android.os.Looper.loop(Looper.java:223)
E/AndroidRuntime(15606): 	at android.app.ActivityThread.main(ActivityThread.java:7664)
E/AndroidRuntime(15606): 	at java.lang.reflect.Method.invoke(Native Method)
E/AndroidRuntime(15606): 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
E/AndroidRuntime(15606): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)
I/Process (15606): Sending signal. PID: 15606 SIG: 9
```

**Comment by jiusanzhou** [comment]
@braxtondiggs thank you for your information, and your solution is inspiring.
I have fixed it in the v1.0.6.

Feel free to reopen the issue if you guys have any problem about this.
